raspiaudio-muse-luxe.yaml

```
@@ -124,6 +181,15 @@ sensor:
           alpha: 0.2
           send_every: 2
       - delta: 0.002
+      - calibrate_polynomial:
+          degree: 3
+          datapoints:
+            - 3.0 -> 0.0
+            - 3.4 -> 20.0
+            - 3.8 -> 50.0
+            - 4.04 -> 80.0
+            - 4.2 -> 100.0
+      - lambda: return std::max(std::min((double)x, 0.0), 100.0);
```

This line will always return 100. It should look like this.
```suggestion
      - lambda: return std::max(std::min((double)x, 100.0), 0.0);
```

---

Right. I messed up the measurements and scale, I'm pushing a fix now.